api/src/test/groovy/uk/gov/dhsc/htbhf/claimant/controller/NewClaimSpec.groovy

```diff
@@ -140,6 +145,13 @@ class NewClaimSpec extends Specification {
         assertValidationResponse(response, "claimant.dateOfBirth", "must not be null")
     }
 
+    private assertAddress(Address actual, AddressDTO expected) {
+        assertThat(actual.addressLine1).isEqualTo(expected.addressLine1)
```

Comment:
I know it will fail if this is the case, but I always like to have a specific assertion to make sure the object is not null first so we can control the error message better.